app/models/account_user.rb

```diff
@@ -7,6 +7,10 @@ class AccountUser < ActiveRecord::Base
   ACCOUNT_OWNER = "Owner".freeze
   ACCOUNT_ADMINISTRATOR = "Business Administrator".freeze
 
+  scope :administrators, lambda {
```

Comment:
This feels a little weird to call it a `scope` since it appears to return `User`s instead of `AccountUser`. Maybe just a class method?